contracts/Registry.vy

```diff
@@ -148,18 +166,29 @@ def _get_rates(_pool: address) -> uint256[MAX_COINS]:
     rates: uint256[MAX_COINS] = empty(uint256[MAX_COINS])
     base_pool: address = self.pool_data[_pool].base_pool
     if base_pool == ZERO_ADDRESS:
-        rate_method_id: Bytes[4] = slice(self.pool_data[_pool].rate_method_id, 28, 4)
+        rate_method_id: bytes32 = self.pool_data[_pool].rate_method_id
 
         for i in range(MAX_COINS):
             coin: address = self.pool_data[_pool].coins[i]
             if coin == ZERO_ADDRESS:
                 break
             if coin == self.pool_data[_pool].ul_coins[i]:
                 rates[i] = 10 ** 18
+            elif rate_method_id == AAVE_RATE_METHOD_ID:
+                rates[i] = 10 ** 18  # aave pegs coins 1:1 w/ underlying asset
```

Comment:
The aave approach isn't unique to Aave, and we _can_ be forward thinking here. What if instead of requiring a specific method ID here (which isn't actually correct, right? they don't have one we can call) we just make the default behavior on a zero-bytes `rate_method_id` be to return 10**18. Saves us writing some non-zero data to storage and easy to understand.

Reply:
Agreed, definitely like removing my convoluted method of having to use a special rate_method_id for aave.